packages/data/src/quotes.yaml

```
@@ -50,7 +50,8 @@
 - quote: S'il pleuvait de la soupe elle/il sortirait avec une fourchette.
 - quote: Si l'intelligence est un bagage, elle/il voyage léger.
 - quote: Si la connerie se mesurait, elle/il servirait de mètre étalon.
-- quote: Si les cons volaient, elle/il serait chef·fe d'escadrille.
+- quote: Si les con·ne·s volaient, elle/il serait chef·fe d'escadrille.
+- quote: Si les con·ne·s étaient en orbite, elle/il arrêterait pas de tourner.
```

Inversé les deux pour garder l'ordre alphabétique

```suggestion
- quote: Si les con·ne·s étaient en orbite, elle/il arrêterait pas de tourner.
- quote: Si les con·ne·s volaient, elle/il serait chef·fe d'escadrille.
```